misk-testing/src/main/kotlin/misk/testing/MockTracingBackendModule.kt

```diff
@@ -6,6 +6,7 @@ import misk.inject.KAbstractModule
 
 class MockTracingBackendModule : KAbstractModule() {
   override fun configure() {
-    bind<Tracer>().to<MockTracer>().asEagerSingleton()
+    bind<MockTracer>().asEagerSingleton()
+    bind<Tracer>().to<MockTracer>()
```

Comment:
needed to split this out to ensure I could inject a MockTracer and get the same underlying instance